test/deep_stacks.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
```

Comment:
`set -euo pipefail`

Reply:
I have some amount of trouble with this setting, though I understand the value. I added it.